process/Development.md

```diff
@@ -21,5 +21,6 @@ You are responsible for shepherding your work through the development process. I
    - make the changes
    - notify the reviewer by tagging them in a comment
 - **Once you have two approvals and one test:**
+- Label your PR **ready**
```

Comment:
I feel like PR can be conflated to be the same as issue. It's inaccurate thought, sure, but if we can prevent it, lets.